include/tvm/runtime/packed_func.h

```diff
@@ -1622,6 +1641,20 @@ inline TVMRetValue PackedFunc::operator()(Args&&... args) const {
   return rv;
 }
 
+template <int i, typename T>
```

Comment:
This should be `size_t` not `int`.  Otherwise it fails to compile with clang.

Reply:
Yeah I found the same thing: https://github.com/apache/tvm/pull/15637...It works with my GCC but not clang